controllers/suite_test.go

```diff
@@ -37,21 +46,28 @@ import (
 // These tests use Ginkgo (BDD-style Go testing framework). Refer to
 // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
 
-var cfg *rest.Config
-var k8sClient client.Client
-var testEnv *envtest.Environment
+var (
+	cfg       *rest.Config
+	k8sClient client.Client // You'll be using this client in your tests.
+	testEnv   *envtest.Environment
+	ctx       context.Context
+	cancel    context.CancelFunc
 
-func TestAPIs(t *testing.T) {
+	kubeconfigName = "template-controller-tests.kubeconfig"
+	kubeconfigPath string
```

Comment:
What is the kubeconfig used for? Do we need it?

Reply:
I used it to debug RBAC issues, but I can actually remove it again